src/Compilers/CSharp/Test/Emit/PDB/PDBUsingTests.cs

```diff
@@ -275,6 +275,86 @@ class C { void M() { } }
 </symbols>");
         }
 
+        [Fact]
+        public void TestTypeAliases1_B()
+        {
+            var text = @"
+using P = string;
+
+class A { void M() { } }
+
+namespace X
+{
+    using Q = int;
+
+    class B { void M() { } }
+
+    namespace Y
+    {
+        using R = char;
+
+        class C { void M() { } }
+    }
+}
+";
+            CompileAndVerify(text, options: TestOptions.DebugDll, targetFramework: TargetFramework.NetFramework).VerifyPdb(@"
```

Comment:
>targetFramework: TargetFramework.NetFramework [](http://example.com/codeflow?start=66&length=45)

nit: Curious, is the TFM needed?

Reply:
yes.  otherwise the referenced symbols come from other dlls, and the specific dlls are checked in the test output.